tests/src/FunctionalJavascript/FieldMapTest.php

```diff
@@ -39,6 +39,12 @@ class FieldMapTest extends WebDriverTestBase {
    *   - Validates saving an mpx media entity and that the field is visible.
    */
   public function testMapCustomField() {
+    $this->container->get('config.factory')
+      ->getEditable('media.settings')
+      ->set('standalone_url', TRUE)
+      ->save(TRUE);
+    $this->container->get('router.builder')->rebuild();
```

Comment:
Curious. What was the thing that needed the rebuild here?

Reply:
The above media setting (FALSE by default for new installs) re-enables the canonical media view. Otherwise, the test fails because /media/1 redirects to /media/1/edit. This is copied from media module tests.

Reply:
Cool, thanks for the explanation!